Firestore/Source/API/FSTUserDataReader.mm

```diff
@@ -351,7 +351,7 @@ - (ParsedUpdateData)parsedUpdateData:(id)input {
     auto parsedEntry = [self parseData:entry context:context.ChildContext(idx)];
     if (!parsedEntry) {
       // Just include nulls in the array for fields being replaced with a sentinel.
-      parsedEntry = NullValue();
+      parsedEntry = nanopb::MakeMessage(NullValue());
```

Comment:
You need to clone the proto to take ownership (maybe we should add a nanopb::CloneMessage or something)